src/WebJobs.Script/Workers/Rpc/RpcWorkerDescription.cs

```diff
@@ -203,6 +203,20 @@ internal void FormatWorkerPathIfNeeded(ISystemRuntimeInformation systemRuntimeIn
                              .Replace(RpcWorkerConstants.RuntimeVersionPlaceholder, DefaultRuntimeVersion);
         }
 
+        internal void FormatArgumentsIfNeeded()
+        {
+            if (Arguments?.Any() == true)
+            {
+                for (int i = 0; i < Arguments.Count; i++)
+                {
+                    if (!string.IsNullOrEmpty(Arguments[i]))
+                    {
+                        Arguments[i] = Arguments[i].Replace(RpcWorkerConstants.WorkerDirectoryPath, WorkerDirectory);
```

Comment:
Might be worth adding a log here to let us know that we did an argument replacement